plugins/inputs/elasticsearch/README.md

```diff
@@ -18,8 +19,12 @@ and optionally [cluster](https://www.elastic.co/guide/en/elasticsearch/reference
   ## within the cluster
   local = true
 
-  ## set cluster_health to true when you want to also obtain cluster level stats
-  cluster_health = false
+  ## set cluster_health to true when you want to also obtain cluster health stats
+  #   cluster_health = false
```

Comment:
OK. I assume you mean the spaces between the # and the comment in 
  #   cluster_stats = false
to 
  # cluster_stats = false

Also, I should take the # off the cluster_heath = false. It was not there before adding it in my version was not intended.